reliquery/storage.py

```diff
@@ -121,14 +121,14 @@ def get_metadata(self, path: StoragePath, root_key: str) -> Dict:
                 "text": [],
                 "html": [],
                 "images": [],
-                "json": [],
+                "jsons": [],
```

Comment:
I think there may have been a miscommunication on my part. We have to be careful changing the names of symbols we have already released. This can break the UI if not it is not updated there as well as if the UI isn't on the latest version.

Reply:
I think we should revert any name changes that aren't related to notebooks